presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Release 335 (XX May 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
+* Return ``VARCHAR`` type when :func:`substr` argument is ``CHAR`` type. (:issue:`3599`)
+* Add a variant of :func:``substring`` that takes a ``CHAR`` argument. (:issue:`3949`)
+
+Security Changes
+----------------
+
+* Add  ``information_schema.role_authorization_descriptors`` table that returns information about what roles where granted to what principals. (:issue:`3535`)
+* Adding schema access rules to file based system access control. (:issue:`3766`)
+
+Hive Connector Changes
+----------------------
+
+* Fix a bug that caused missing column statistics after ``INSERT`` to new table. (:issue:`2469`)
+* Add support for ``information_schema.role_authorization_descriptors`` table when ``hive.security=sql-standard``.  (:issue:`3535`)
+* Allow non-lowercase column names in ``system.sync_partition_metadata`` procedure. This can be enabled
+  by passing ``case_sensitive=false`` when invoking the procedure. (:issue:`3431`)
+* Make caching work when coordinator is secured. (:issue:`3874`)
+* Prevent caching from turning off in case of intermittent network failures. (:issue:`3874`)
+* Make caching and HDFS impersonation mutually exclusive. (:issue:`3913`)
+* Add ``hive.cache.ttl`` and ``hive.cache.disk-usage-percentage`` cache properties. (:issue:`3840`)
+* Schedule work on nodes with cached data. (:issue:`3922`)
+* Add support for ``UNIONTYPE``.  This is mapped to ``ROW`` containing a ``tag`` field and a field for each data type in the union. For
```

Comment:
Is there docs for this so we can link to it instead?

Reply:
I don't think so. @dain?

Reply:
Nope.  We don't have any docs on data type mappings for Hive.

Reply:
So lets leave this as is. Should I create an issue to add that? @electrum, @mattsfuller and myself already talked about breaking up the Hive connector docs into multiple docs as well as having some sort of standard setup for each connector documentation. This could be one of the things each connector doc has..